pkg/pillar/cmd/zedrouter/flowstats.go

```diff
@@ -652,6 +721,7 @@ func DNSMonitor(bn string, bnNum int, ctx *zedrouterContext, status *types.Netwo
 			if switched && dnslayer == nil {
 				dnssys[bnNum].Lock()
 				checkDHCPPacketInfo(bnNum, packet, ctx)
```

Comment:
Should this return true if it was DHCP to avoid checking for DAD probe?

Reply:
done